export/cloud_datastore_admin.py

```diff
@@ -0,0 +1,87 @@
+# Copyright 2018 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Request handler for exporting backups."""
+
+import datetime
+import httplib
+import json
+import logging
+
+from core.domain import acl_decorators
+
+from google.appengine.api import app_identity
+from google.appengine.api import urlfetch
+import webapp2
+
+
+class ExportToCloudDatastoreHandler(webapp2.RequestHandler):
+
+    @acl_decorators.can_perform_cron_tasks
+    def get(self):
+        access_token, _ = app_identity.get_access_token(
+            'https://www.googleapis.com/auth/datastore')
+        app_id = app_identity.get_application_id()
+        timestamp = datetime.datetime.utcnow().strftime('%Y%m%d-%H%M%S')
+
+        output_url_prefix = self.request.get('output_url_prefix')
+        assert output_url_prefix and output_url_prefix.startswith('gs://')
+
+        # Look for slash in the portion of the bucket URL that comes
+        # after 'gs://'. If not present, then only a bucket name has been
+        # provided and we append a trailing slash.
+        if '/' not in output_url_prefix[5:]:
```

Comment:
You'll still want to do something about the "5:". Maybe extract 'gs://' into a named constant.

Reply:
done